openmetadata-ui/src/main/resources/ui/src/components/Entity/EntityHeaderTitle/EntityHeaderTitle.component.tsx

```diff
@@ -97,24 +97,26 @@ const EntityHeaderTitle = ({
         <div
           className="d-flex gap-3 items-center"
           data-testid="entity-header-title">
-          <Typography.Text
-            className={classNames(
-              'entity-header-display-name',
-              displayNameClassName,
-              'm-b-0 subheading text-md font-medium'
-            )}
-            data-testid="entity-header-display-name"
-            ellipsis={{ tooltip: true }}
-            style={{ color: color ?? TEXT_COLOR }}>
-            {stringToHTML(displayName || name)}
-            {openEntityInNewPage && (
-              <IconExternalLink
-                className="anticon vertical-baseline m-l-xss"
-                height={14}
-                width={14}
-              />
-            )}
-          </Typography.Text>
+          <Tooltip placement="bottom" title={stringToHTML(name)}>
```

Comment:
```suggestion
          <Tooltip placement="bottom" title={stringToHTML(displayName || name)}>
```
Also better to compute it once and pass it to tooltip and rendering